c/models/aggregator.cc

```diff
@@ -33,7 +33,9 @@ static PKArgs args_aggregate(
   },
   "aggregate",
 
-R"(aggregate(dt, min_rows, n_bins, nx_bins, ny_bins, nd_max_bins, max_dimensions, seed, progress_fn, nthreads, double_precision)
+R"(aggregate(dt, min_rows=500, n_bins=500, nx_bins=50, ny_bins=50,
```

Comment:
Call `dt` -> `frame` instead, so that there is no confusion with `import datatable as dt` ?

Reply:
Done